[2023-12-12T02:07:33.305+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airbyte_rtgs_data_processing_airflow_dag.airbyte_check_sync_rtgs_journal_entries manual__2023-12-12T02:07:10.325039+00:00 [queued]>
[2023-12-12T02:07:33.317+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airbyte_rtgs_data_processing_airflow_dag.airbyte_check_sync_rtgs_journal_entries manual__2023-12-12T02:07:10.325039+00:00 [queued]>
[2023-12-12T02:07:33.318+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-12T02:07:33.348+0000] {taskinstance.py:1382} INFO - Executing <Task(AirbyteJobSensor): airbyte_check_sync_rtgs_journal_entries> on 2023-12-12 02:07:10.325039+00:00
[2023-12-12T02:07:33.358+0000] {standard_task_runner.py:57} INFO - Started process 361 to run task
[2023-12-12T02:07:33.372+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'airbyte_rtgs_data_processing_***_dag', 'airbyte_check_sync_rtgs_journal_entries', 'manual__2023-12-12T02:07:10.325039+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/rtgs_data_processing.py', '--cfg-path', '/tmp/tmp0zrl5bpf']
[2023-12-12T02:07:33.378+0000] {standard_task_runner.py:85} INFO - Job 102: Subtask airbyte_check_sync_rtgs_journal_entries
[2023-12-12T02:07:33.443+0000] {task_command.py:416} INFO - Running <TaskInstance: airbyte_rtgs_data_processing_airflow_dag.airbyte_check_sync_rtgs_journal_entries manual__2023-12-12T02:07:10.325039+00:00 [running]> on host 86ddea81acd7
[2023-12-12T02:07:34.116+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='airbyte_rtgs_data_processing_***_dag' AIRFLOW_CTX_TASK_ID='airbyte_check_sync_rtgs_journal_entries' AIRFLOW_CTX_EXECUTION_DATE='2023-12-12T02:07:10.325039+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-12T02:07:10.325039+00:00'
[2023-12-12T02:07:34.130+0000] {base.py:73} INFO - Using connection ID '***-call-to-airbyte' for task execution.
[2023-12-12T02:07:34.225+0000] {airbyte.py:80} INFO - Waiting for job 34 to complete.
[2023-12-12T02:08:34.258+0000] {base.py:73} INFO - Using connection ID '***-call-to-airbyte' for task execution.
[2023-12-12T02:08:34.346+0000] {airbyte.py:75} INFO - Job 34 completed successfully.
[2023-12-12T02:08:34.347+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-12-12T02:08:34.359+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=airbyte_rtgs_data_processing_***_dag, task_id=airbyte_check_sync_rtgs_journal_entries, execution_date=20231212T020710, start_date=20231212T020733, end_date=20231212T020834
[2023-12-12T02:08:34.405+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-12T02:08:34.806+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
